.github/actions/create-jira-issue/entrypoint.sh

```diff
@@ -18,7 +20,8 @@ curl --request POST \
     "description": "${GITHUB_SERVER_URL}/${GITHUB_REPOSITORY}/actions/runs/${GITHUB_RUN_ID}: ${JIRA_ISSUE_DESCRIPTION}",
     "assignee": {
       "id": "${JIRA_ASSIGNEE_ID}"
-    }
+    },
+  "customfield_10008": ${sprint_id}
```

Comment:
```suggestion
  "customfield_10008": "${sprint_id}"
```

Reply:
I'd like to see a test for the case where there is no active sprint.
I think it will be okay with above modification.

Reply:
@mathewseby Can you just fall back to putting it in the backlog when there is no active sprint?

Reply:
> I think it will be okay with above modification.

Sorry, the Modification is not working. 
Modified script with a condition to check for the active sprint, now if there are no active sprint tickets will be created in the backlog.

Reply:
Thanks!
I also confirmed that it doesn't work.

```
{"errorMessages":["Number value expected as the Sprint id."],"errors":{"customfield_10008":"The Sprint (id) must be a number"}}
```

>Modified script with a condition to check for the active sprint, now if there are no active sprint tickets will be created in the backlog.

👍  Left some sugesstions. Please check!